core/modules/dashboard/includes/block.news.inc

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @file
+ * Dashboard block displaying BackdropCMS.org news items.
+ */
+
+class DashboardNewsBlock extends Block {
+  /**
+   * {@inheritdoc}
+   */
+
+  public function __construct($plugin_name, array $data) {
+    parent::__construct($plugin_name, $data);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Backdrop News');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getAdminTitle() {
+    return t('Backdrop News');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getAdminPreview() {
+    return t('Displays a news feed from BackdropCMS.org.');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getContent() {
+    $feed = backdrop_http_request('https://backdropcms.org/api/notifications');
+    $links = array();
+    if ($feed->code === '200') {
+      $news = json_decode($feed->data);
+      foreach ($news->notifications as $key => $notifications) {
+        $links[] = '<p>' . l(
+            $notifications->notification->title,
+            $notifications->notification->url
+          ) . ' - ' . $notifications->notification->summary . '</p>';
+      }
+    }
+
+    $build = array(
+      '#theme' => 'dashboard_panel__news',
```

Comment:
I dont see this theme function actually implemented? You'd need a `hook_theme()` entry as well. And as a matter of fact, if we're adding a theme function, we should just pass the value of `$news` to the theme function, and build the whole output in the theme function, so site builders can override the output.